src/web/screens/teamMembers.tsx

```diff
@@ -28,21 +30,61 @@ function TeamMembers() {
   const onTeamMembersUpdated = useCallback(() => {
     reloader.revalidate();
   }, [reloader]);
+  const [showStatusPopup, setShowStatusPopup] = useState<boolean>(false);
+  const [statusPopup, setStatusPopup] = useState<StatusNotificationType>();
+  const [popupDuration, setPopupDuration] = useState<number>(3000);
+
+  const handleErrorPopup = (e: Error) => {
+    const hash = Object.hasOwn(e, 'errorHash') ? `: (${(e as ApiError).errorHash})` : '';
+    setStatusPopup({
+      type: 'Error',
+      message: `${e.message}${hash}`,
+    });
+    setShowStatusPopup(true);
+    setPopupDuration(10000);
+    throw new Error(e.message);
+  };
+
+  const handleSuccessPopup = (message: string) => {
+    setStatusPopup({
+      type: 'Success',
+      message,
+    });
+    setShowStatusPopup(true);
+    setPopupDuration(3000);
+  };
 
   const handleAddTeamMember = async (formData: InviteTeamMemberForm) => {
-    await InviteTeamMember(formData, participant!.id);
+    const response = await InviteTeamMember(formData, participant!.id);
+    if (response.statusText === 'Created') {
+      handleSuccessPopup('Team member added.');
+    }
     onTeamMembersUpdated();
   };
 
   const handleRemoveTeamMember = async (userId: number) => {
-    await RemoveUser(userId);
-    onTeamMembersUpdated();
+    try {
+      const response = await RemoveUser(userId);
+      if (response.statusText === 'OK') {
+        handleSuccessPopup('Team member removedfdg dfghdfg hdfgh dfgh drf hdf gh dfg hdf gh.');
```

Comment:
I guess this is just placeholder?

Reply:
Oops.  That's embarrassing.  I was testing long text alignment.  This is fixed.